.github/workflows/build.yml

```diff
@@ -36,6 +36,13 @@ jobs:
         uses: supabase/setup-cli@v1
         with:
           version: 1.93.0
+      - name: Apply Supabase Migrations
+        run: |
+          npm install -g supabase
```

Comment:
the install is not required since, it's done by line 36

Reply:
Done